Sources/MapboxCoreNavigation/CoreFeedbackEvent.swift

```diff
@@ -2,8 +2,8 @@ import Foundation
 import MapboxDirections
 import Polyline
 
-class CoreFeedbackEvent: Hashable {
-    var id = UUID()
+public class CoreFeedbackEvent: Hashable, Codable {
```

Comment:
Remember to add class and method documentation for any public class that’s intended to be used by the application.